vllm/model_executor/model_loader/utils.py

```diff
@@ -88,6 +88,14 @@ def initialize_model(
 def process_weights_after_loading(
     model: nn.Module, model_config: ModelConfig, target_device: torch.device
 ) -> None:
+    if getattr(model, "process_weights_after_loading_already_called", False):
+        # In case `process_weights_after_loading` is called multiple times
+        # we'll skip it at later times
+        logger.warning(
```

Comment:
Should this really be warning? Seems more like a debug statement as it is intentional. Also this will be very noisy calling for each layer

Reply:
oh, can change this to debug, yeah it's doesn't have to warn here I think.